Week 1/Weekly Practice Problems (Week 1)/0709: To Lower Case/judah_silva.cpp

```diff
@@ -0,0 +1,11 @@
+class Solution {
+public:
+    string toLowerCase(string s) {
+        for (int i = 0; i < s.length(); i++) {
+            if (s[i] >= 65 && s[i] <= 90) {
```

Comment:
We can substitute 65 and 90 with 'A' and 'Z' respectively! Either way is fine, but it's easier to remember the letters rather than their ASCII counterparts for competitions.
```suggestion
            if (s[i] >= 'A' && s[i] <= 'Z') {
```